apps/cowswap-frontend/src/modules/hooksStore/dapps/AirdropHookApp/constants.ts

```diff
@@ -17,6 +18,21 @@ export const AIRDROP_OPTIONS = [
       },
     },
 
+    decimals: 18,
+  },
+  {
+    name: 'YYY',
+    dataBaseUrl: 'todo/',
+    addressesMapping: {
+      [SupportedChainId.SEPOLIA]: '',
+    },
+    tokenMapping: {
+      [SupportedChainId.SEPOLIA]: {
+        ...COW[SupportedChainId.SEPOLIA],
+        address: '',
+      },
+    },
```

Comment:
Think that we just need one example of airdrop. After the review, we could also comment to let it there as an example.